templates/includes/webp-img.jade

```diff
@@ -0,0 +1,43 @@
+mixin webp-img(src, title, alt, picture_style, img_style)
+  
+  //- remove img src path suffix 
+  - var path_nosuffix = src.replace(/[.][a-z]+/,"");
+  //- get absolute path from the src path and remove suffix
+  - var absolute_path = (path.resolve("./contents"+src.match(/\/media.+/, "")));
+  - var absolute_nosuffix = absolute_path.replace(/[.][a-z]+/,"");
+
+  //- for blogs use picture style
+  //- if the style is not supplied then calculate img ratio from image dimensions
```

Comment:
Extend the note. This is a "hack"/workaround and not a long term solution because the code assumes that if there is no function argument specified for `picture_style`, then we assume that it is not a blog post and use some default classes specified here in the mixin.